apps/docs/app/features/routes/ressurser/design-tokens/SpacingTokens.tsx

```diff
@@ -119,23 +54,29 @@ const SpacingTokensTable = ({ ...props }: SpacingTokenTableProps) => {
           </Tr>
         </Thead>
         <Tbody>
-          {spacingTokens.map((token) => (
-            <Fragment key={token.key}>
+          {Object.entries(spacingSizes).map(([key, token]) => (
```

Comment:
By just using the tokens object directly, we don't have to specify all tokens explicitly. Doing this for the typography tokens as a bit harder, because there was a bit of extra meta information required that was not present in the tokens.